src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -880,12 +881,19 @@ protected boolean getAcls(Checkpoint checkpoint) throws DfException {
         groups;
     protected Checkpoint groupsCheckpoint;
 
+    private FeedType feedType;
+    private boolean prevError;
+    protected boolean caughtException;
+
     protected GroupTraverser() {
-      this(Checkpoint.full());
+      this(Checkpoint.full(), FeedType.FULL);
+      prevError = false;
```

Comment:
Bump: move these to the other constructor.

Reply:
Done